src/MPS.jl

```diff
@@ -204,7 +267,7 @@ Base.adjoint(tn::T) where {T<:AbstractMPO} = T(adjoint(Ansatz(tn)), form(tn))
 """
     Base.rand(rng::Random.AbstractRNG, ::Type{MPS}; n, maxdim, eltype=Float64, physdim=2)
 
-Create a random [`MPS`](@ref) Tensor Network.
+Create a random [`MPS`](@ref) Tensor Network where all tensors are left-canonical (MixedCanonical(Site(0))).
```

Comment:
Should be right-canonical right?

```suggestion
Create a random [`MPS`](@ref) Tensor Network where all tensors are right-canonical (orthogonality center at site 1).
```